### ISSUE TITLE
Can you provide the pretraining log of UCF101? 

### ISSUE THREAD

**Issue by Randle-Github** [Mark12Ding/FAME#5]
Hi, I confronted a problem with your code but transfer to some other datasets that the contrastive loss could hardly drop. I wonder if it is a normal phenomenon that such loss could not demonstrate the results of the model or some other problems. If possible, can you provide your log with UCF101 or just provide some details of the loss curve? Thanks.

**Comment by Mark12Ding** [comment]
Well, the loss **does** **drop** **slightly** at the pretraining stage due to the harder objective in our setting. And the loss curve doesn't indicate whether the model is pre-trained well or not. Instead, the results of the downstream tasks can prove it.
